### PR TITLE
speedup eli5.explain_.. functions

### PR DESCRIPTION
**Pull request by kmike**
Features with zero coefficients are no longer shown. For one example it reduced explain_prediction time from 20s to 40ms: most coefficients are zero when linear model coefficients are multiplied by instance vector.

### CODE REVIEW

**Comment by lopuhin** [comment]
Neat! 👍 
It should mostly speed up explain_prediction, right?

**Comment by kmike** [comment]
Yeah; speedup for explain_weights will only show up if coefficients are sparse.
